### ISSUE TITLE
`--show-skipped` tests option

### ISSUE THREAD

**Issue by Sigmanificient** [Snaipe/Criterion#502]
Hi, it would be nice to add a option to show the number of skipped tests.
This would avoid setting verbose to check for skipped tests.

```
[====] Synthesis: Tested: 0 | Passing: 0 | Failing: 0 | Crashing: 0 
```

would become 
```
[====] Synthesis: Tested: 0 | Passing: 0 | Failing: 0 | Crashing: 0 | Skipped: 1
```
when the flag `--show-skipped` is set.

**Comment by drawbu** [comment]
Yeah, I feel like this should be part of Criterion by default, skipped tests are a nice feature, but the fact that we don't see anything make it much less pleasant to use...
